src/sections/unsereTeams.html

```diff
@@ -0,0 +1,84 @@
+<div class="relative z-10 py-10 px-8 md:py-20 md:px-10 pt-20 xl:pt-52 xl:px-36 ojemba-oblique3 2xl:grid justify-center">
```

Comment:
I think you should start different sections in the same way to keep your code consistent. Some sections begin with the section tag, while others start with div tag. Make sure to use the same approach everywhere.